core-tests/shared/src/test/scala/zio/internal/ExecutorSpec.scala

```diff
@@ -35,9 +37,9 @@ object TestExecutor {
   }
 }
 
-class ExecutorSpec extends Specification {
+object SpecDescription {
```

Comment:
You can drop this entire object, the `is` function is a specs2 artefact.

Reply:
should I maybe somewhere put description what actually tests does?